app/helpers/search_bar_helper.rb

```diff
@@ -0,0 +1,27 @@
+module SearchBarHelper
+  def mount_search_bar(
+    id,
+    options = {
```

Comment:
In ruby, unlike JS, if you pass an option hash it will override this default one so you would have to pass all of the options. instead, you can use [kwargs](https://robots.thoughtbot.com/ruby-2-keyword-arguments) to allow setting defaults for any option not provided by the user.